packages/sui-widget-embedder/src-react/Widget.js

```diff
@@ -9,30 +9,36 @@ export default class Widget extends Component {
     children: PropTypes.element,
     node: PropTypes.string,
     i18n: PropTypes.object,
-    domain: PropTypes.object
+    domain: PropTypes.object,
+    isVisible: PropTypes.bool
+  }
+
+  static defaultProps = {
+    isVisible: true
   }
 
   componentDidMount() {
-    const {node: selector, children, i18n, domain} = this.props
+    const {node: selector, children, i18n, domain, isVisible} = this.props
     const node = document.querySelector(selector)
 
     if (!node) {
       return console.warn(`[Widget] unable find the selector ${selector}`) // eslint-disable-line
     }
 
-    ReactDOM.render(
-      <Context.Provider
-        value={{
-          i18n,
-          domain
-        }}
-      >
-        <ProviderLegacy i18n={i18n} domain={domain}>
-          {children}
-        </ProviderLegacy>
-      </Context.Provider>,
-      node
-    )
+    isVisible &&
+      ReactDOM.render(
+        <Context.Provider
```

Comment:
Doesn't this fit in one line?

Reply:
Prettier auto formats may seem inconsistent sometimes.